app/models/mission_control/jobs/recurring_task.rb

```diff
@@ -1,7 +1,7 @@
 class MissionControl::Jobs::RecurringTask
   include ActiveModel::Model
 
-  attr_accessor :id, :job_class_name, :command, :arguments, :schedule, :last_enqueued_at, :queue_name, :priority
+  attr_accessor :id, :job_class_name, :command, :arguments, :schedule, :last_enqueued_at, :next_time, :queue_name, :priority
```

Comment:
How would this get populated? 🤔

Reply:
Hmm. Either I missed something or things have changed since I created this PR 😄 

It should be coming from [SolidQueue::RecurringTask](https://github.com/rails/solid_queue/blob/51c75bec01c8c53f2fe573822f0158e8b62030aa/app/models/solid_queue/recurring_task.rb#L52C5-L54C8), but it seems l needed to add [ecd58a9](https://github.com/rails/mission_control-jobs/pull/165/commits/ecd58a90ddf391fcba0059e0602f201801576c63) to make that happen.